.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,30 +1,14 @@
-ARE YOU PLANNING TO CHANGE THE STYLE GUIDE'S PUBLIC API? If so, please make sure you've first gone through the RFC approval process by opening a new issue and using the provided template.
+** Please provide as much detail as possible, but feel free to remove irrelevant sections **
 
-IMPORTANT
-=========
-
-Please ensure that, when eventually merging this PR to master, you copy the completed "Commit Message For Review" section below into your final commit body. This allows us to review our release notes, validating that they're clear enough for consumers to follow.
-
-Don't forget that what's written here forms the upgrade guide for consumers, and upgrading can easily become a long and arduous process. To counter this, please be as descriptive as possible, explaining the reasoning behind the change, and showing a clear usage example or migration guide where appropriate. Since there are many different consumers, changes should be documented from the perspective of the style guide itself, so avoid making specific reference to the application you're working on.
-
-
-Major release template
-======================
-
-
-## Commit Message For Review
+{{ Reason for change }}
 
 BREAKING CHANGE:
 
 {{ Description of breaking API change }}
 
 RFC URL:
```

Comment:
Worth adding conditions for sections like these? eg. `RFC URL (for api changes only):` and `BREAKING CHANGE (if any):

Reply:
Yeah, it's tricky. I'm trying to minimise the amount of careful editing you need to perform on this.

Reply:
That's why I added the first line to this file about removing irrelevant sections.